### ISSUE TITLE
[bug] .provide() throws error in Sage 10

### ISSUE THREAD

**Issue by ouun** [roots/bud#1233]
### Agreement

- [X] This is not a duplicate of an existing issue
- [X] I have read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
- [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

### Expected Behavior

Using

```
.provide({
  jquery: '$',
})
```

in `bud.config.js` in Sage 10.

### Actual Behavior

Bud breaks with error `Internal Error: Error: TypeError: v.forEach is not a function`.

### Steps To Reproduce

Add

```
.provide({
  jquery: '$',
})
```

in `bud.config.js` in Sage 10. The error has been confirmed by @joshuafredrickson at Roots Slack.

### version

5.5.0

### What package manager are you using?

yarn classic

### version

1.22.17

### Are you using pnpm?

- [ ] yes
- [X] no

### Logs

```zsh
yarn run v1.22.17
$ bud dev
Internal Error: Error: TypeError: v.forEach is not a function
    at DevCommand.make (/sites/sage/wp-content/themes/sage/node_modules/@roots/bud/lib/cjs/cli/commands/base.js:71:19)
    at async DevCommand.execute (/sites/sage/wp-content/themes/sage/node_modules/@roots/bud/lib/cjs/cli/commands/build.js:209:9)
    at async DevCommand.validateAndExecute (/sites/sage/wp-content/themes/sage/node_modules/clipanion/lib/advanced/Command.js:67:26)
    at async Cli.run (/sites/sage/wp-content/themes/sage/node_modules/clipanion/lib/advanced/Cli.js:236:24)
[process] › ℹ  info      beforeExit
[process] › ℹ  info      exit
✨  Done in 1.44s.
```

### Configuration

```zsh
/**
 * @typedef {import('@roots/bud').Bud} bud
 *
 * @param {bud} app
 */
module.exports = async (app) => {
  app
    /**
     * Application entrypoints
     *
     * Paths are relative to your resources directory
     */
    .entry({
      app: ['@scripts/legacy', '@styles/legacy', '@styles/app']
    })

    /**
     * The following specifies that $ is the default jQuery export:
     * Now, in any module in our application, we can invoke jQuery with $. There is no need to import it.
     */
    .provide({
      jquery: '$',
    })

    /**
     * These files should be processed as part of the build
     * even if they are not explicitly imported in application assets.
     */
    .assets('images')

    /**
     * These files will trigger a full page reload
     * when modified.
     */
    .watch('resources/views/**/*', 'app/**/*')

    /**
     * Target URL to be proxied by the dev server.
     *
     * This is your local dev server.
     */
    .proxy('https://example.test')

    /**
     * Development URL
     */
    .serve({
      url: 'https://example.test:3000',
    });
};
```

**Comment by kellymears** [comment]
Thanks for the issue. I'll look into this and set up some unit tests.

In the meantime: `bud.provide` is a pretty thin wrapper around `webpack-provide-plugin`. You can set this option like so:

```ts
bud.extensions
  .get("webpack-provide-plugin")
  .setOption("$", "jquery")
```

Here it is in a `tap` call so it's chainable:

```ts
bud.tap(bud => bud.extensions.get("webpack-provide-plugin").setOption("$", "jQuery"))
```

I put it in a `bud.tap` call so it's drop-in (`bud.extensions` doesn't chain). But that's optional.

**Comment by kellymears** [comment]
Just realized you might want `$` and `jQuery`:

```ts
bud.extensions.get("webpack-provide-plugin").setOptions({
  $: "jquery",
  jQuery: "jquery",
})
```

**Comment by kellymears** [comment]
I snuck a fix for this into #1229

**Comment by retlehs** [comment]
related topic on roots discourse: https://discourse.roots.io/t/sage-10-adding-jquery/22476

**Comment by retlehs** [comment]
fixed in [5.6.0](https://github.com/roots/bud/releases/tag/v5.6.0) 🎉
